build/screener/screener.config.js

```diff
@@ -10,7 +10,7 @@ require('tsconfig-paths').register({
 
 // https://github.com/screener-io/screener-runner
 module.exports = {
-  projectRepo: 'stardust-ui/react',
+  projectRepo: 'microsoft/fluent-ui-react',
```

Comment:
This PR only creates our new screener project.  We need this commit separate so we can ensure our new set of baseline images are correct.  No other changes are made here to guarantee our first `microsoft/fluent-ui-react` images are exact matches to the latest approved master `@stardust-ui/react` images.